### PR TITLE
Updating LICENSE section of readme.md

### PR DESCRIPTION
**Pull request by kshithijiyer**

### CODE REVIEW

**Review by varyvol — APPROVED** [review]

File: README.md
Comment (oleg-nenashev) [inline README.md]:
```suggestion
[MIT License]: https://github.com/jenkinsci/jenkins/blob/master/LICENSE.txt
```

**Review by oleg-nenashev — APPROVED** [review]
I agree there is no need to repeat the license. I suggest using the Blob layout so that users see the license terms clarification from GitHub Web UI. https://github.com/jenkinsci/jenkins/blob/master/LICENSE.txt

File: README.md
Comment (kshithijiyer) [reply README.md]: Done

**Comment by kshithijiyer** [comment]
> I agree there is no need to repeat the license. I suggest using the Blob layout so that users see the license terms clarification from GitHub Web UI. https://github.com/jenkinsci/jenkins/blob/master/LICENSE.txt

Done

**Comment by oleg-nenashev** [comment]
Will merge tomorrow if no negative feedback

**Comment by kshithijiyer** [comment]
> Will merge tomorrow if no negative feedback

Should we merge it now?

**Comment by oleg-nenashev** [comment]
I was waiting for CI to restore. No worries, it should land in the next release

**Comment by oleg-nenashev** [comment]
CI is dead, so I will just go ahead and merge it. there is no CI coverage for Markdown anyway

**Comment by oleg-nenashev** [comment]
Thanks @kshithijiyer !

**Pull request merged**
